terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,55 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                = "search-service" # this must match the stack name the service deploys into
+  name_prefix               = "${local.stack_name}-${var.environment}"
+  service_name              = "restricted-word-web"
+  container_port            = "3000" # default node port required here until prod docker container is built allowing port change via env var
+  docker_repo               = "restricted-word-web"
+  lb_listener_rule_priority = 96
+  lb_listener_paths         = ["/admin/restricted-word","/admin/restricted-word/.*%"]
```

Comment:
This path won't work `"/admin/restricted-word/.*%"`, regex syntax is different on aws use `"/admin/restricted-word/*"`

Reply:
Done